packages/nodes-base/nodes/Twist/CommentDescription.ts

```diff
@@ -0,0 +1,623 @@
+		type: 'string',
+		default: '',
+		displayOptions: {
+			show: {
+				operation: [
+					'get',
+					'remove',
+				],
+				resource: [
+					'comment',
+				],
+			},
+		},
+		required: true,
+		description: 'The ID of the comment',
+	},
+	/* -------------------------------------------------------------------------- */
+	/*                                 comment:getAll                             */
+	/* -------------------------------------------------------------------------- */
+	{
+		displayName: 'Workspace ID',
+		name: 'workspaceId',
+		type: 'options',
+		typeOptions: {
+			loadOptionsMethod: 'getWorkspaces',
+		},
+		default: '',
+		displayOptions: {
+			show: {
+				operation: [
+					'getAll',
+				],
+				resource: [
+					'comment',
+				],
+			},
+		},
+		required: true,
+		description: 'The ID of the workspace.',
+	},
+		{
+		displayName: 'Thread ID',
+		name: 'threadId',
+		type: 'string',
+		default: '',
+		displayOptions: {
+			show: {
+				operation: [
+					'getAll',
+				],
+				resource: [
+					'comment',
+				],
+			},
+		},
+		required: true,
+		description: 'The ID of the channel.',
+	},
+	{
+		displayName: 'Return All',
+		name: 'returnAll',
+		type: 'boolean',
+		displayOptions: {
+			show: {
+				resource: [
+					'comment',
+				],
+				operation: [
+					'getAll',
+				],
+			},
+		},
+		default: false,
+		description: 'If all results should be returned or only up to a given limit.',
+	},
+	{
+		displayName: 'Limit',
+		name: 'limit',
+		type: 'number',
+		displayOptions: {
+			show: {
+				resource: [
+					'comment',
+				],
+				operation: [
+					'getAll',
+				],
+				returnAll: [
+					false,
+				],
+			},
+		},
+		typeOptions: {
+			minValue: 1,
+			maxValue: 100,
+		},
+		default: 50,
+		description: 'How many results to return.',
+	},
+	{
+		displayName: 'Filters',
+		name: 'filters',
+		type: 'collection',
+		placeholder: 'Add Field',
+		default: {},
+		displayOptions: {
+			show: {
+				resource: [
+					'comment',
+				],
+				operation: [
+					'getAll',
+				],
+			},
+		},
+		options: [
+			{
+				displayName: 'As IDs',
+				name: 'as_ids',
+				type: 'boolean',
+				default: false,
+				description: 'If enabled, only the ids of the comments are returned.',
+			},
+						{
+				displayName: 'Starting object index',
+				name: 'from_obj_index',
+				type: 'number',
+				default: 0,
+				description: 'Limit comments starting at the specified object index.',
+			},
+			{
+				displayName: 'Ending object index',
+				name: 'to_obj_index',
+				type: 'number',
+				default: 50,
+				description: 'Limit comments ending at the specified object index.',
+			},
+						{
+				displayName: 'Order By',
+				name: 'order_by',
+				type: 'options',
+								options: [
+					{
+						name: 'ASC',
+						value: 'ASC',
+					},
+										{
+						name: 'DESC',
+						value: 'DESC',
+					},
+				],
+				default: 'ASC',
+				description: 'The order of the comments returned one of DESC or ASC.',
```

Comment:
Also for all others.

```suggestion
				description: 'The order of the comments returned - one of DESC or ASC.',
```